db/seeds/playbooks/spooky.seeds.rb

```diff
@@ -0,0 +1,273 @@
+      a miss, you’ll get magical backlash.'
+    },
+    {
+      name: 'Jinx',
+      rating: :weird,
+      type: 'Moves::Rollable',
+      playbook_id: @spooky.id,
+      six_and_under: 'On a miss, the Keeper holds 2 over you to be used in the 
+      same way.',
+      seven_to_nine: 'On a 7-9, hold 1.',
+      ten_plus: 'On a 10+, hold 2.',
+      description: 'You can encourage coincidences to occur, the way you want. 
+      When you jinx a target, roll +Weird.  Interfere with a hunter, giving 
+      them -1 forward.  Help a hunter, giving them +1 forward, by interfering 
+      with the enemy.  Interfere with what a monster, minion, or bystander is 
+      trying to do.  Inflict 1-harm on the target due to an accident.  The target 
+      finds something you left for them.  The target loses something that you 
+      will soon find.'
+    }
+  ].each do |move|
+    Move.find_or_create_by!(move)
+  end
+  
+    #####
+    # Gear
+    #####
+  
+    # Note: Spooky have custom weapons
+  [
+    {
+      name: '.38 revolver',
+      armor: 2,
+      tag_list: %w[close reload loud]
+    },
+    {
+      name: '9mm',
+      armor: 2,
+      tag_list: %w[close loud]
+    },
+    {
+      name: 'Hunting rifle',
+      armor: 2,
+      tag_list: %w[far loud]
+    },
+    {
+      name: 'Shotgun',
+      armor: 3,
+      tag_list: %w[close messy]
+    },
+    {
+      name: 'Big knife',
+      armor: 1,
+      tag_list: %w[hand]
+    }
+  ].each do |gear_attrs|
+    gear = Gear.find_or_create_by!(
+      name: gear_attrs[:name],
+      playbook: @spooky
+    )
+    gear.update!(gear_attrs)
+  end
+
+    #####
+    # Improvements
+    #####
+  [
+    {
+      description: 'Get +1 Weird, max +3',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 3,
+      rating: :weird,
+      playbook: @spooky
+    },
+    {
+      description: 'Get +1 Charm, max +2',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 2,
+      rating: :charm,
+      playbook: @spooky
+    },
+    {
+      description: 'Get +1 Cool, max +2',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 2,
+      rating: :cool,
+      playbook: @spooky
+    },
+    {
+      description: 'Get +1 Sharp, max +2',
+      type: 'Improvements::RatingBoost',
+      stat_limit: 2,
+      rating: :sharp,
+      playbook: @spooky
+    },
+    {
+      description: 'Take another Spooky move',
+      type: 'Improvements::PlaybookMove',
+      playbook: @spooky,
+      stat_limit: 0
+    },
+    {
+      description: 'Take another Spooky move',
+      type: 'Improvements::PlaybookMove',
+      playbook: @spooky,
+      # Used to differentiate (and initially hide this) from the identical improvement above.
+      stat_limit: -1
+    },
+    {
+      description: 'Change some, or all, your dark side tags',
+      playbook: @spooky,
+    },
+    {
+      description: 'Get a mystical library, like the Expert’s haven option',
+      type: 'Improvements::AnotherMove',
```

Comment:
This should be 'Improvements::HavenMove'

Reply:
We should make an issue for this move too, I'll make it in the morning. Haven Move currently does not let you specify the move.